cypress/integration/plugins/security-analytics-dashboards-plugin/3_alerts.spec.js

```diff
@@ -71,6 +75,9 @@ describe('Alerts', () => {
     cy.get('[data-test-subj="superDatePickerCommonlyUsed_Today"]').click({
       force: true,
     });
+
+    // wait for the table entries to be ready, needed for slower systems
```

Comment:
Just curious if there isnt a way around this wait. Isnt there a selector we can wait for instead?